src/Apps/FeatureAKG/Components/Feature.tsx

```diff
@@ -173,24 +176,38 @@ const Section: React.FC<SectionProps> = props => {
   )
 }
 
-const VideoSection: React.FC<{ src: string }> = props => {
-  // Constrain the height, add 1px for the border
+const Video: React.FC<{ src: string }> = props => {
   return (
-    <Box mb="-40px" height="591px">
-      <SectionSeparator mt="4" />
-      <Box textAlign="center">
-        <Video src={props.src} />
-      </Box>
-    </Box>
+    <StyledVideo
+      autoPlay
+      loop
+      muted
+      playsInline
+      controls={false}
+      src={props.src}
+    />
   )
 }
 
-const Video: React.FC<{ src: string }> = props => {
+const StyledVideo = styled("video")`
+  width: 100%;
+  height: auto;
+`
+
+const ImageSection: React.FC<{ src: string }> = props => {
   return (
-    <video autoPlay loop muted playsInline controls={false} src={props.src} />
+    <BorderedSection>
+      <Image src={props.src} width="100%" height="auto" />
```

Comment:
These gifs are well below the fold, but they are meant to be responsive. Do you know of any techniques for lazyloading images that don't have a fixed height?

Reply:
Hmmm... So you tried the `ResponsiveImage` component and that didn't work?